<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LYE</class>
 <widget class="QDialog" name="LYE">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LYE</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                        stop:0 #FFFFFF, stop:1 #AAAAAA);</string>
  </property>
  <widget class="QWidget" name="login_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>400</height>
    </rect>
   </property>
   <widget class="QPushButton" name="loginn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font-size: 15px; font-stretch: normal;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Log in </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordd">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #DCDCDC, stop:1 #AAAAAA);
border: 2px solid white; 
    border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #DCDCDC, stop:1 #AAAAAA);
border: 2px solid white; 
    border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_message1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;; 
color:red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'Snell Roundhand', cursive; font-size: 25px; font-stretch: normal;
background-color: rgba(255, 255, 255, 0);
</string>
    </property>
    <property name="text">
     <string>      Your name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'Snell Roundhand', cursive; font-size: 25px; font-stretch: normal;
background-color: rgba(255, 255, 255, 0);
</string>
    </property>
    <property name="text">
     <string>    Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prev">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> font-size: 15px; font-stretch: normal;
background-color: rgba(255, 255, 255, 0);
</string>
    </property>
    <property name="text">
     <string>return</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_message2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;; 
color:green;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="LYE_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #FFFFFF, stop:1 #AAAAAA);
border: none;
font-weight: bold;
text-shadow: 2px 2px 2px rgba(0, 0, 0, 0.5);
color: #222222;
border-radius: 25px;</string>
   </property>
   <property name="text">
    <string>LYE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
